src/vcpkg/base/files.cpp

```diff
@@ -582,6 +582,25 @@ namespace vcpkg::Files
         return result;
     }
 
+    void Filesystem::create_best_link(const fs::path& to, const fs::path& from, std::error_code& ec)
+    {
+        this->create_hard_link(to, from, ec);
```

Comment:
why do we want a hard link here? it seems like we should prefer a symlink, at least to me.

Reply:
This is actually old code from a previous implementation and can be simply removed -- the reason to prefer a hard link is that it does not become invalidated by move or remove